backup_and_restore/application_backup_and_restore/aws-sts/oadp-aws-sts.adoc

```diff
@@ -0,0 +1,43 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-aws-sts"]
+= Backing up applications on AWS STS using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-aws-sts-backing-up-applications
+
+toc::[]
+
+You install the OpenShift API for Data Protection (OADP) with Amazon Web Services (AWS) by installing the OADP Operator. The Operator installs link:https://{velero-domain}/docs/v{velero-version}/[Velero {velero-version}].
+
+include::snippets/oadp-mtc-operator.adoc[]
+
+You configure {aws-short} for Velero, create a default `Secret`, and then install the Data Protection Application. For more details, see xref:../../..//backup_and_restore/application_backup_and_restore/installing/oadp-installing-operator.adoc#oadp-installing-operator-doc[Installing the OADP Operator].
+
+To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
+
+You can install OADP on an AWS Security Token Service (AWS STS) cluster manually. Amazon AWS provides AWS Security Token Service (AWS STS) as a web service that enables you to request temporary, limited-privilege credentials for users. You use STS to provide trusted users with temporary access to resources via API calls, your AWS console or the AWS command line interface (CLI).
```

Comment:
Use the attribute for "Security Token Service". Looks like that attribute is: {sts-first}

See: https://redhat-documentation.github.io/supplementary-style-guide/#_product_names_and_version_references